packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -48,14 +53,73 @@ class TimelineData {
   /// The current node in the tree structure of GPU duration events.
   TimelineEvent _gpuEventNode;
 
+  final HeapPriorityQueue<TraceEventWrapper> _cpuTraceHeap =
+      HeapPriorityQueue(traceComparator);
+
+  final HeapPriorityQueue<TraceEventWrapper> _gpuTraceHeap =
+      HeapPriorityQueue(traceComparator);
+
+  static int traceComparator(TraceEventWrapper a, TraceEventWrapper b) {
```

Comment:
nit: move this to the TraceEventWrapper itself. Seems reasonable in general.

Reply:
donel